src/main/java/com/ibm/eventstreams/connect/mqsource/JMSReader.java

```diff
@@ -97,26 +97,27 @@ public void configure(Map<String, String> props) {
         String sslCipherSuite = props.get(MQSourceConnector.CONFIG_NAME_MQ_SSL_CIPHER_SUITE);
         String sslPeerName = props.get(MQSourceConnector.CONFIG_NAME_MQ_SSL_PEER_NAME);
         String topic = props.get(MQSourceConnector.CONFIG_NAME_TOPIC);
+        String csp = props.get(MQSourceConnector.CONFIG_NAME_MQ_CSP);
 
         int transportType = WMQConstants.WMQ_CM_CLIENT;
         if (connectionMode != null) {
             if (connectionMode.equals(MQSourceConnector.CONFIG_VALUE_MQ_CONNECTION_MODE_CLIENT)) {
                 transportType = WMQConstants.WMQ_CM_CLIENT;
-            } 
+            }
             else if (connectionMode.equals(MQSourceConnector.CONFIG_VALUE_MQ_CONNECTION_MODE_BINDINGS)) {
                 transportType = WMQConstants.WMQ_CM_BINDINGS;
-            } 
+            }
             else {
                 log.error("Unsupported MQ connection mode {}", connectionMode);
                 throw new ConnectException("Unsupported MQ connection mode");
-            }               
+            }
         }
 
         try {
             mqConnFactory = new MQConnectionFactory();
             mqConnFactory.setTransportType(transportType);
             mqConnFactory.setQueueManager(queueManager);
-            mqConnFactory.setBooleanProperty(WMQConstants.USER_AUTHENTICATION_MQCSP, true);
+            mqConnFactory.setBooleanProperty(WMQConstants.USER_AUTHENTICATION_MQCSP, Boolean.parseBoolean(csp));
```

Comment:
This needs to default to 'true'. The new code defaults to 'false'.

Reply:
done